docs/Operators.md

```diff
@@ -885,47 +887,72 @@ Equation definitions:
 `h` - hidden gate
 `t` - time step (t-1 means previous time step)
 `Xi` - input tensor
-`W[izrfcoh]` - W parameter weight matrices for the corresponding gates
-`R[izrfcoh]` - R parameter weight matrices for the corresponding gates
-`Wb[izrfcoh]` - W parameter bias vectors for the corresponding gates
-`Rb[izrfcoh]` - R parameter bias vectors for the corresponding gates
+`W[izrfcohp]` - W parameter weight matrices for the corresponding gates
+`R[izrfcohp]` - R parameter weight matrices for the corresponding gates
+`Wb[izrfcohp]` - W parameter bias vectors for the corresponding gates
+`Rb[izrfcohp]` - R parameter bias vectors for the corresponding gates
 `ReLU(X)` - max(X, 0)
 `tanh` - hyperbolic tangent of X
 `sigmoid(X)` - 1 / (1 + e^-X)
 `[C|H]` - Cell/Hidden state
 
 - Equations:
-  `relu`
+  `simple(relu)`
   - Ht = ReLU(Wi*Xt + Ri*Ht-1 + Wbi + Rbi)
```

Comment:
We may add the clip to all the formulas: ReLU(clip(Wi*Xt + Ri*Ht-1 + Wbi + Rbi))

Reply:
Clip is optional and disabled as default. There are comments on the clip attribute.